app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,101 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+$gray-light : #777;
```

Comment:
For the tutorial its fine but in DocFox core we pull these kinds of magic values from css / scss / sass variables ... lets us keep the corporate image and reuse the same magic values. Also true for spacing

Reply:
Dont worry about any of the styling comments I make here. They are there for your own information for when you work on DocFox core